README.md

```diff
@@ -45,111 +50,104 @@ for your learners to practice in.
 6.  Make sure the repository is public, and click on "Create repository from template".
 You will be redirected to your new copy of the workshop template.
 
-If you experience a problem, please re-try;
-if the problem persists,
-please [get in touch](#getting-and-giving-help).
+If you experience a problem, please [get in touch](#getting-and-giving-help).
+
+## Customizing Your Website (Required Steps)
+
+There are two ways of customizing your website. You can either:
 
-## Customizing Your Website
+- edit the files directly in GitHub using your web browser
+- clone the repository on your computer and update the files locally
+
+### Updating the files on GitHub in your web browser
 
 1.  Go into your newly-created repository,
     which will be at `https://github.com/your_username/YYYY-MM-DD-site`.
     For example,
     if your username is `gvwilson`,
     the repository's URL will be `https://github.com/gvwilson/2016-12-01-miskatonic`.
 
-3.  Ensure you are on the gh-pages branch by clicking on the branch under the drop 
+3.  Ensure you are on the gh-pages branch by clicking on the branch under the drop
```

Comment:
The default settings for templates mean that only the gh-pages branch is in the new repo and the header space looks quite different from the image below now, so maybe we don't need this or at least update the image?
![select-gh-pages-branch](https://user-images.githubusercontent.com/10656079/80003329-dad78000-848e-11ea-9040-d68c1dfacb03.png)